zerver/worker/queue_processors.py

```diff
@@ -480,6 +482,17 @@ def get_bot_api_client(self, user_profile):
 
     def get_bot_handler(self, service):
         # type: (Service) -> Any
+
+        # Convert unicode service.name to string.
+        str_service_name = unicodedata.normalize('NFKD', service.name).encode('ascii', 'ignore')
+
+        # Assert if this service is present in EMBEDDED_BOT_SERVICES.
+        assert EmbeddedBotService(str_service_name) in EMBEDDED_BOT_SERVICES
+
+        for embedded_bot_service in EMBEDDED_BOT_SERVICES:
+            if embedded_bot_service.name == service.name:
+                assert embedded_bot_service.IS_ALLOWED is True
```

Comment:
Leave off the `is True`; that's not normal Python style and makes the code more confusing.